docs/workshops/kubecon_azure_track.md

```diff
@@ -23,11 +27,16 @@ To follow along using this workshop, simply go through the steps detailed in thi
 
 When you see `eks-node-viewer` use `aks-node-viewer` instead.
 
-## Adjusted Instructions
+> Note: if you ever end up needing to use the extended log command to look back over a longer period of time, make sure its using the `kube-system` namespace like follows:
+> ```bash
+> kubectl -n karpenter logs -f deployment/karpenter --all-containers=true --since=20m
```

Comment:
--all-containers should not be necessary anymore

Reply:
I'm gonna leave this in, since it mirrors the `aws` docs. Will look at removing in the future.